bundles/http/Citrix/Canvas-LMS/metadata.json

```diff
@@ -0,0 +1,48 @@
+{
+  "type" : "HTTP",
+  "vendor" : "Citrix",
+  "id" : "e80ec039-95c1-4c66-bb94-a6cb7a1652fb",
+  "trackingUuid" : "4fde839a-f215-4f57-a0ef-dd16d84216d2",
+  "title" : "Canvas LMS",
+  "description" : "Deploy the Canvas LMS integration to view the course and its related details as a student, and as a teacher to create a course announcement and view the course members and grades.",
+  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/TEGJf2vhv8wexUsRk0l7kfyJlLMcC7R2.png",
+  "masVersion" : "1.93.4.40cd82fa6dc8acc40d248f3d650d1bc3f60b6197",
+  "categories" : [ ],
+  "created" : "2021-06-14T12:40:51",
+  "supportsOAuthForActions" : true,
+  "i18nLanguages" : [ "en" ],
+  "apps" : [ {
+    "uuid" : "344aeaf7-ed94-4bc1-8eb3-548b99afeb88",
+    "trackingUuid" : "ed4d4bc4-972a-4242-b596-4fded910e6d5",
+    "title" : "My Grades",
+    "numberOfEvents" : 1,
+    "action" : true
+  }, {
+    "uuid" : "4d72294f-b482-40d1-8cac-9bab09164078",
+    "trackingUuid" : "cc15833a-deca-4dac-845c-8b2a02f265e7",
+    "title" : "Create Course Announcement",
+    "numberOfEvents" : 0,
+    "action" : true
+  }, {
+    "uuid" : "6172f674-07f2-4517-ab17-c2e7d6b0c938",
+    "trackingUuid" : "fc407347-2137-4ab4-9633-db7e4d01bb9a",
+    "title" : "Teacher's View",
+    "numberOfEvents" : 0,
+    "action" : true
+  }, {
+    "uuid" : "fe515648-eda4-4e8f-9c6e-0a9d66d40e5e",
+    "trackingUuid" : "e0971412-6bce-4dfa-8006-dc7fff4821dd",
+    "title" : "My Courses",
+    "numberOfEvents" : 3,
+    "action" : true
+  } ],
+  "vaResolvers" : [ ],
+  "tags" : [ ],
```

Comment:
Please add helpLink.

Reply:
Updated in this commit `Updated Bundle Review comments, category, helpURL`.